### ISSUE TITLE
Stuff to do!

### ISSUE THREAD

**Issue by zahid** [MichaelAng/todo-client#6]
It's necessary to complete this example application

**Comment by zahid** [comment]
Need:
- [x] application level component - https://github.com/MichaelAng/todo-client/pull/8
- [x] Flatten the circle shell scripts
- [ ] CRUD operations for todo models in the API - in progress, zm
- Unit tests
    - [ ] Client
    - [ ] API
- Integration tests
    - [ ] Client
    - [ ] API
- Linting
    - [ ] Client
    - [ ] API
- [ ] Postman collection to use with the api - in progress, zm
- [x] Runs via docker & docker-compose - https://github.com/MichaelAng/todo-client/pull/8
- [ ] Documentation
- [x] Docker cloud setup - https://github.com/MichaelAng/todo-client/pull/8
- ~Kubernetes setup~ - OUT OF SCOPE
- Hook up the client to connect to the api
- [ ] Get some seed to dos in there
- [ ] Ensure NGINX is forwarding requests to the API from the client

Questions:
1. Should this also connect to a database like MongoDB? If we want to persist CRUD operations (like add and delete and update a post) we'll need a db @MichaelAng

**Comment by zahid** [comment]
Can I get push rights to this repo?

**Comment by MichaelAng** [comment]
Yes it should.   For the Source environment we should have it connect to a mongoDB container.  In the future, we should hook it up with mLab
